### ISSUE TITLE
contact not receiving unsecured sms

### ISSUE THREAD

**Issue by LJOaKz9uXj02wu** [WhisperSystems/Signal-Android#6873]
I have:
- [x] searched open and closed issues for duplicates
- [x ] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
When I text a certain contact using unsecured sms they no longer receive my text messages.  They were receiving them up until a few days ago.  When I send them the message it puts a check mark next to it like it was sucessfuly sent, and I don't receive any type of error or indication it failed.
  When I text a different contact with unsecured sms they still receive them though. 

### Steps to reproduce
-Send unsecured sms to certain contact

**Actual result:** They never receive it 
**Expected result:** They would receive it or at least give me indication it failed

### Device info

**Device:** samsung SM-G920P (zerofltespr)
**Android version:** 7.0 (G920PVPS4DQF1, NRD90M.G920PVPS4DQF1)
**Signal version:** 4.8.1

### Link to debug log

https://gist.github.com/anonymous/5c33882724e2f5acdcb7e7700929e9ed

**Comment by moxie0** [comment]
If you get one check mark, it means that we've handed the SMS to the OS, and the OS has reported that it was delivered to your wireless carrier. It seems unlikely that your issues are related to Signal, when you use a different SMS app on your device to text the same contact, do they receive it?

**Comment by LJOaKz9uXj02wu** [comment]
yeah it works when i use the stock sms app.  I just noticed that also when I try to send unsecured sms to my signal contacts the same thing happens.  They do receive them though if I use stock sms app.

https://gist.github.com/anonymous/02cbff08f8486b5a1d2ebb3b2b35bb84

**Comment by moxie0** [comment]
Everything looks normal in those logs. We hand the SMS to the OS, and it reports that it has been sent. We don't really have any control over anything else that happens beyond the OS interface we get to the carrier. Perhaps your carrier doesn't like E164 numbers.

**Comment by moxie0** [comment]
@LJOaKz9uXj02wu is the number formatted E164 in your contacts? Does your registered Signal number have the same country code as your current SIM?
